docker/build.sh

```diff
@@ -39,13 +39,13 @@ shift $((OPTIND - 1))
 export ARCH=${ARCH:-x86_64}
 export PYVER=${PYVER:-3.6}
 export PYV=${PYVER/./}
-export CUDA_VERSION=${CUDA_VERSION:-11}
+export CUDA_VERSION=${CUDA_VERSION:-110}
```

Comment:
Would there be any benefit of handling it similarly to PYVER and allowing user to provide `11.1` instead of `111`?

Reply:
Why not.

Reply:
Done